explain paused actions for info clients (#215)
valkey-io/valkey-doc: commands/info.md

*   `total_watched_keys`: Number of watched keys. Added in Valkey 8.0.
*   `total_blocking_keys`: Number of blocking keys.
*   `total_blocking_keys_on_nokey`: Number of blocking keys that one or more clients that would like to be unblocked when the key is deleted.

> *   `paused_actions`: The current paused actions of the instance: "all" means all clients will be paused,
>     "write" means clients executing write commands will be paused,
>     and "none" means no clients will be paused. Added in Valkey 8.1.
> *   `paused_timeout_milliseconds`: The remaining time of the paused actions. Added in Valkey 8.1.

Reviewer: "will be" is the future. They are paused now, right?

```suggestion
*   `paused_actions`: The current paused actions of the instance: "all" means all clients are paused,
    "write" means clients executing write commands are paused,
    and "none" means no clients are paused. Added in Valkey 8.1.
*   `paused_timeout_milliseconds`: The remaining time of the paused actions. Added in Valkey 8.1.
```

Reviewer 2: It also includes upcoming clients, which might have been why it might be future tense. But I think your suggestion is better.
